### PR TITLE
init symfony 7 support

### PR DESCRIPTION
**Pull request by Sakonokode**
@Nek- Hello, voila une PR qui permet le bump vers symfony 7 et php 8.0

### CODE REVIEW

File: tests/Melodiia/Doctrine/DoctrineDataStoreTest.php [hunk 1]
```diff
@@ -11,57 +11,66 @@
...
 class DoctrineDataStoreTest extends TestCase
 {
-    public function testItIsAMelodiiaDataStore()
+    /** @var Prophet */
+    private $prophet;
+
+    public function setUp(): void
+    {
+        $this->prophet = new Prophet();
+    }
```
Comment (Nek-) [inline tests/Melodiia/Doctrine/DoctrineDataStoreTest.php:27]: Can you please use this package instead ? https://packagist.org/packages/phpspec/prophecy-phpunit

**Review by Nek- — CHANGES_REQUESTED** [review]
Hello,

This looks great, but is it really compatible with Symfony 5? wow nice!

I'm sorry I saw you transformed all tests already but can you integrate the PHPUnit trait instead ? 🙏

File: tests/Melodiia/Doctrine/DoctrineDataStoreTest.php (hunk 1 above)
Comment (Sakonokode) [reply tests/Melodiia/Doctrine/DoctrineDataStoreTest.php:27]: I don't know about Symfony 5, i will test it and complete your suggestions asap :+1:

**Review by Nek- — APPROVED** [review]

**Comment by Nek-** [comment]
Thank you very much @Sakonokode !

**Pull request merged**